packages/integration-karma/test/light-dom/basic/index.spec.js

```diff
@@ -0,0 +1,40 @@
+import { createElement, setFeatureFlagForTest, LightningElement } from 'lwc';
+
+import Test from 'x/test';
+
+describe('Basic Light DOM', () => {
+    it('should render properly', () => {
+        setFeatureFlagForTest('ENABLE_LIGHT_DOM_COMPONENTS', true);
+        const elm = createElement('x-test', { is: Test });
+        document.body.appendChild(elm);
+
+        expect(elm.shadowRoot).toBeNull();
+        expect(elm.firstChild.innerText).toEqual('Hello, Light DOM');
+        setFeatureFlagForTest('ENABLE_LIGHT_DOM_COMPONENTS', false);
+    });
+    it('should render to Shadow DOM when feature is disabled', () => {
+        setFeatureFlagForTest('ENABLE_LIGHT_DOM_COMPONENTS', false);
+        expect(() => createElement('x-test', { is: Test })).toThrowError();
```

Comment:
Assert against the error message itself. This way you can ensure that the code throws the error you expect and not something completely unrelated.